bridge_adaptivity/bridge_lti/provider.py

```diff
@@ -77,6 +77,17 @@ def get_collection_collectiongroup_engine(collection_id, group_slug):
         raise Http404('Bad launch_url collection ID.')
 
     collection_group = CollectionGroup.objects.filter(slug=group_slug).first()
+
+    if collection_group is None:
+        raise Http404('Not correctly configured launch URL. Could not find group with slug `{}`'.format(
+            group_slug
+        ))
+
+    if collection not in collection_group.collections.all():
+        raise Http404('Not correctly configured launch URL. Collection with ID `{}` not in group with slug `{}`'.format(
```

Comment:
text typos:
`Not correctly configured launch URL. Collection with the ID: `{}` not in the group with the slug `